### ISSUE TITLE
HTML tags formatting

### ISSUE THREAD

**Issue by mchyb** [9e15/guidelines#11]
To keep our code unitary among the team we should specify a common style of formatting HTML tags.

### The proposal:  

When they don't exceed the maximum characters in line:
```
<tag-name first-attribute="attribute-value" second-attribute="attribute-value"></tag-name>
```
When they exceed the maximum characters in line:
```
<tag-name 
  first-attribute="attribute-value"
  second-attribute="attribute-value"
  third-attribute="attribute-value">
</tag-name>
```
The attributes' values should always be put between double quotation marks.

When naming tags' identifiers and classes, hyphens and lower case letters style should be preffered:

```
<tag-name 
  id="identifier-name"
  class="class-name">
</tag-name>
```

**Comment by kownacki** [comment]
LGTM

**Comment by merlinnot** [comment]
LGTM

**Comment by mchyb** [comment]
Resolved by #18
